src/main/java/frc/robot/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -26,4 +26,9 @@ public double getEncoderValue2() {
   public void periodic() {
     elevatorSystem.updateInputs();
   }
+
```

Comment:
I don't think we need this after all. Should remove and try - the `periodic` call should be enough.